src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/mailActionFields.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import SelectMembers from 'app/components/selectMembers';
+import SelectControl from 'app/components/forms/selectControl';
+import {Organization, Project} from 'app/types';
+import {IssueAlertRuleAction, MailActionTargetType} from 'app/types/alerts';
+import {PanelItem} from 'app/components/panels';
+import space from 'app/styles/space';
+
+type Props = {
+  project: Project;
+  organization: Organization;
+  disabled: boolean;
+  loading: boolean;
+  action: IssueAlertRuleAction;
+  onChange: (action: IssueAlertRuleAction) => void;
+};
+
+interface OptionRecord {
+  value: string;
+  label: string;
+}
+
+class MailActionFields extends React.Component<Props> {
+  handleChange = (
+    attribute: 'targetType' | 'targetIdentifier',
+    e: OptionRecord & {[key: string]: any}
```

Comment:
Can we either rename `e` or maybe destructure the properties you need?

Reply:
Done, thanks!